src/types.jl

```diff
@@ -541,10 +546,12 @@ function ReplicaSystem(;
     k_converted = convert_k_units(T, k, energy_units)
     K = typeof(k_converted)
 
-    replicas = Tuple(System{D, G, T, CU, A, AD, PI, SI, GI, C, V, B, NF,
+    constraints = ()
```

Comment:
This could be a keyword argument to the `ReplicaSystem` constructor, and add it to the docstring.